utils/build/docker/python_otel/flask-poc-otel/app.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+opentelemetry-instrument gunicorn -w 1 -b 0.0.0.0:7777 --access-logfile - app:app -k gevent
```

Comment:
Just curious why are we using gunicorn? Can we just use the default flask development server.